cylc/flow/scripts/clean.py

```diff
@@ -120,10 +121,17 @@ def get_option_parser():
     parser.add_option(
         '--timeout',
         help=("The number of seconds to wait for cleaning to take place on "
-              "remote hosts before cancelling."),
+              r"remote hosts before cancelling. Default: %default."),
```

Comment:
Nice to document the defaults, I have no idea why optparse does not do this by "default".

We are already defining our own "help formatter", looks like we could override `expand_default` to add a "Default: %default" bit onto the end of every option where a default is set rather than having to add it to each option manually. (for future work).